### ISSUE TITLE
Add GaussDB(for MySQL) endpoint for ap-southeast-1

### ISSUE THREAD

**Issue by ik8svip** [huaweicloud/terraform-provider-huaweicloud#3252]
### Current Terraform Version

```bash
Terraform v1.5.3
on linux_amd64
+ provider registry.terraform.io/huaweicloud/huaweicloud v1.53.0
```

### Use-cases

HCL code:

```hcl
# https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/data-sources/gaussdb_mysql_flavors
data "huaweicloud_gaussdb_mysql_flavors" "flavors" {
  region                 = "ap-southeast-1"
  availability_zone_mode = "multi"       # multi|single
  engine                 = "gaussdb-mysql" # "gaussdb-mysql"
  version                = "8.0"                      # "8.0" 
}
```
Error
```bash
╷
│ Error: Error fetching flavors for gaussdb mysql, error: Get "https://gaussdb.ap-southeast-1.myhuaweicloud.com/v3/a029682174324eb295191391efd6a9ce/flavors/gaussdb-mysql?version_name=8.0&availability_zone_mode=multi": dial tcp: lookup gaussdb.ap-southeast-1.myhuaweicloud.com on 127.0.0.53:53: no such host
│ 
│   with module.rds.data.huaweicloud_gaussdb_mysql_flavors.flavors,
│   on ../../modules/rds/data.tf line 8, in data "huaweicloud_gaussdb_mysql_flavors" "flavors":
│    8: data "huaweicloud_gaussdb_mysql_flavors" "flavors" {
│ 
╵
```

### Attempted Solutions

Try find the some docs, looks like thiere's no endpoint for `ap-southeast-1`, https://developer.huaweicloud.com/intl/en-us/endpoint

### Proposal

Add support for each region.

**Comment by ShiChangkuo** [comment]
@ik8svip the endpoint of GaussDB(for MySQL) in ap-southeast-1 region is different that other regions.
As I tested, you should add the following configuration in **provider** block.

```hcl
provider "huaweicloud" {
  ...
  endpoints = {
    gaussdb = "https://gaussdbformysql.ap-southeast-1.myhuaweicloud.com"
  }
}
```

**Comment by ShiChangkuo** [comment]
@ik8svip it will be fixed in v1.63.0 at the end of this month.
